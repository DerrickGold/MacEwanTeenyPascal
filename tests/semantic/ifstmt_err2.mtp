(*Test if condition with non-Boolean value*)
const test := 'error';
var a, b: integer;

begin
	if test then
	   write('Hello')
	;
end.