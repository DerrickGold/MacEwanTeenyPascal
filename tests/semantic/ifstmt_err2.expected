
Semantic Error [line 6]: value: "sc1_test"
	Invalid conditional type: Expected type(s) 'Boolean'; found 'String' instead.
